Components/cards.css

```diff
@@ -0,0 +1,156 @@
+@import url(/Styles/variables-and-fonts.css);
+
+
+/* <!-- Card Basic --> */
+
+.card-container {
+    box-sizing: border-box;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    height: fit-content;
+    width: 15rem;
+    margin: 1rem;
+    border-radius: 0.5rem;
+    box-shadow: 0px 5px 5px 1px gray;
+    justify-content: flex-start;
+}
+
+.card-container:hover{
+    box-shadow: 0px 5px 10px 1px gray;
+    border: 1px solid gray;
+}
+
+
+.card-img {
+    width: 100%;
+    height: 60%;
+    border-radius: inherit;
+}
+
+.card-title {
+    margin: 0.5rem 0.5rem;
+}
+
+.card-subtitle {
+    margin: 0rem 0.5rem;
+    color: gray;
+}
+
+.card-para {
+    margin: 1rem 0.5rem;
+    font-size: small;
+    color: gray;
+}
+
+.card-actions-container {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0.3rem 0.5rem;
+}
+
+.card-button {
+    margin: 0 0rem;
+    cursor: pointer;
+    color: var(--primary-color);
+    background-color: unset;
+    border: none;
+    font-weight: bold;
+    font-size: small;
+    border-radius: 0.5rem;
+    padding: 0.5rem 0.3rem;
+}
+
+.card-button:hover, .card-icon:hover { 
+    background: rgba(79, 189, 186, 0.2)
+}
+
+.card-icon {
+    margin: 0 0.1rem;
+    cursor: pointer;
+    color: var(--primary-color);border-radius: 0.5rem;
+    padding: 0.2rem;
+}
+
+/* <!-- Card with Badge --> */
+
+.card-badge {
+    position: absolute;
+    top: 1rem;
+    box-sizing: inherit;
+    border-radius: 0.5rem;
+    background-color: red;
+    color: white;
+    padding: 0.2rem 1rem;
```

Comment:
you can use variables for padding and margins as well if you want it really helped me.